pkg/packagekit/updates.jsx

```diff
@@ -251,19 +252,18 @@ function updateItem(info, pkgNames, key) {
         type = (
             <>
                 <Tooltip id="tip-severity" content={ secSeverity || _("security") }>
-                    <span className={iconClasses} />
+                    <span aria-label={secSeverity || _("security")} className={iconClasses + 'severity-icon'} />
```

Comment:
you need to put there space before `severity-icon` as now it creates something like `class="fooseverity-icon"` and not `class="foo severity-icon"`.